stable/traefik/values.yaml

```diff
@@ -59,6 +59,20 @@ acme:
   email: admin@example.com
   staging: true
   logging: false
+  # Configure a Let's Encrypt certificate to be managed by default.
+  # This is the only way to request wildcard certificates (works only with dns challenge).
+  domains:
+    enabled: false
+    # List of sets of main and (optional) SANs to generate for
+    # for wildcard certificates see https://docs.traefik.io/configuration/acme/#wildcard-domains
+    domainsList:
```

Comment:
I think the examples in this section should be commented out - this is the norm for examples, and also the listed example domains are added to the active configuration.

Reply:
Finally some feedback, thank you @grugnog!

Changed pushed.